src/frameworks/index.ts

```diff
@@ -537,25 +538,24 @@ ${
       if (bootstrapScript) await writeFile(join(functionsDist, "bootstrap.js"), bootstrapScript);
 
       // TODO move to templates
-
+      const mergedFrameworksBackend = {
+        ...functionsDirectoryFrameworksBackend,
+        ...frameworksBackend,
```

Comment:
I think `functionsDirectoryFrameworksBackend` should come after so it can override the default values?

Reply:
I'm thinking that the user could set the memory to a higher value, in that case we should not override it.